compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/readonly-object-method-calls-mutable-lambda.expect.md

```diff
@@ -45,15 +45,7 @@ function Component(props) {
   posts.push(t0);
   const count = posts.length;
   foo(count);
-  let t1;
-  if ($[1] !== posts) {
-    t1 = <>{posts}</>;
-    $[1] = posts;
-    $[2] = t1;
-  } else {
-    t1 = $[2];
-  }
-  return t1;
+  return <>{posts}</>;
```

Comment:
this is weird. technically correct - posts will always invalidate, so no point memoizing the jsx - but i want to double-check why the memo is getting removed here